autonomx/main.cpp

```diff
@@ -59,15 +59,15 @@ int main(int argc, char *argv[]) {
     const char* applicationName = "autonomX";
     const char* applicationVersion = "0.1.1";
     const char* organizationName = "Xmodal";
-    const char* extensionName = "atnx";
 
     // global settings
     QCoreApplication::setApplicationName(applicationName);
     QCoreApplication::setApplicationVersion(applicationVersion);
     QCoreApplication::setOrganizationName(organizationName);
 
     Settings* settings = new Settings();
-    settings->setValue("global/extensionName", extensionName);
+    settings->setValue("global/extensionName", "atnx");
+    settings->setValue("global/repoUrl", "https://github.com/Xmodal/autonomX");
```

Comment:
We should put this value in a constant in order to make the code more readable and maintainable.